tests/run-travis.sh

```diff
@@ -11,14 +21,14 @@ function run_packages_tests {
 			export NAME=$(basename $(pwd))
 
 			if [ ! -e tests/php/travis-can-run.sh ] || tests/php/travis-can-run.sh; then
-				if [ "$DO_COVERAGE" == "true" ]; then
-					composer install
+				if [ "$DO_COVERAGE" == "true" ] && [[ "$NAME" != "codesniffer" ]]; then
```

Comment:
With the changes you made below, this can no longer be reached with DO_COVERAGE=true. Line 135 should have already exited.

Reply:
Yeah, missed that. Thank you for spotting it! Cleaned it in 4000664